src/E2EEBanner.module.css

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2023 New Vector Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.e2eeBanner {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 16px;
```

Comment:
the font size in the [figma](https://www.figma.com/file/AjvSw4lI9iGO5aTe4WAl4r/Element-Call?type=design&node-id=3842-74107&mode=design&t=yo0QxoFmpsS4st3E-0) desings is 12px. This should be the same than the font size of the "By clicking "Go", you agree to our [Terms and conditions](https://call.ems.host/_matrix/consent?v=1.0)" Label.